pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -170,42 +170,42 @@ func getDefaultCNINetwork(confDir string, binDirs []string) (*cniNetwork, error)
 		if strings.HasSuffix(confFile, ".conflist") {
 			confList, err = libcni.ConfListFromFile(confFile)
 			if err != nil {
-				klog.Warningf("Error loading CNI config list file %s: %v", confFile, err)
+				klog.InfoS("Error loading CNI config list file", "confFile", confFile, "err", err)
```

Comment:
Assume that confFile is just a path to a file
```suggestion
				klog.InfoS("Error loading CNI config list file", "path", confFile, "err", err)
```